lib/shopify-cli/messages/messages.rb

```diff
@@ -300,6 +310,8 @@ module Messages
               organization_not_found: "Cannot find a partner organization with that ID",
               partners_notice: "Please visit https://partners.shopify.com/ to create a partners account",
             },
+            first_party: "Are you working on a 1P (1st Party) app?",
```

Comment:
@tiffanytse I made a small change to the wording here. Previously we agreed on `Are you building a 1P (1st Party) app?` but that doesn't make as much sense when using the `connect` command to connect to an existing app.